### ISSUE TITLE
Faster debugging of broken cluster

### ISSUE THREAD

**Issue by johnSchnake** [vmware-tanzu/sonobuoy#1608]
Conformance-lite was meant to be a way to debug a broken cluster quickly but it is actually failed at that since it runs so many tests and even though the tests _pass_ quickly, they can fail slowly causing the same problem as before.

We can follow up on #1604 to help this, but we should also have another plan.

**Comment by johnSchnake** [comment]
I tried splitting up the 'fast' tests into smaller chunks and maxing out the parallelization. I have 37 plugins with just 7 tests each and each one running in parallel.

Locally I couldn't get this to run terribly well because its so resource constrained on my single machine. In the happy path (locally) it took ~10m to run all the tests up from 6m originally.

However, I set up an EKS cluster with 8 worker nodes and:

 - In the unhappy path (I set KUBE_TEST_REPO to an invalid value so all pod pulls would error) it took 31m to finish.

 - In the happy path they all passed in about 3m

**Comment by johnSchnake** [comment]
With larger worker nodes I tried to run again and found that same ~30m timeframe. I changed the nodes to be the aws large nodes instead and also set an extra env var to tell ginkgo to use 7 parallel threads (I noticed in the logs that sometimes it ran with just 2, presumably due to resource constraints).

In this case the unhappy path repeated above only took 18m.

I also confirmed that using `sonobuoy logs -f |rg FAILED` was giving me the test failures as they came through for even earlier research on the failures.
